Fix leaderboard behavior in scores of not happened wods

joaopcanario opened the pull request:
Copying the behavior of Crossfit Games leaderboard for WODs that aren't opened yet. In this case, the table cell has to be empty.

joaopcanario committed:
Add CNAME

joaopcanario committed:
Remove CNAME

joaopcanario committed:
Merge remote-tracking branch 'upstream/master'

joaopcanario committed:
Merge remote-tracking branch 'upstream/master'

joaopcanario committed:
Fix leaderboard behavior in scores of not happened wods

cristianossd commented on src/App.js:
```diff
@@ -134,13 +134,17 @@ class App extends Component {
-                                <small>({ score.scoreDisplay })</small>
-                              </td>
-                            )
+                            if (score.rank === 0) {
```
```js
score.rank ?
  return (
    <td>
      <strong>{ this.ordinalSuffix(score.rank) }</strong>
      <br />
      <small>({ score.scoreDisplay })</small>
    </td>
  )
  : return (<td />)
```

joaopcanario replied:
When I added the ternary operator a Syntax error (unexpected token) is raised on 'return' token.

joaopcanario replied:
Fixed! Please check the last commit

joaopcanario committed:
Merge remote-tracking branch 'upstream/master'

joaopcanario committed:
Fix PR #14

joaopcanario committed:
Revert "Fix leaderboard behavior in scores of not happened wods"

This reverts commit 6f0b07b686dfab6f566562da557f5d22b686235e.

joaopcanario committed:
Fix ternary operator

In suggested improvement on PR #14 this error is raised on `return` token:
`Syntax error: Unexpected token`

joaopcanario committed:
Add PR #14 suggestion
